lineapy/api/models/linea_artifact.py

```diff
@@ -79,9 +76,10 @@ class LineaArtifact:
     """name of the artifact"""
```

Comment:
nitpick but nice to fix: i think its more common to have the comment above the parameter and not below right?

Reply:
Not sure we should move this, I think this is for the RTD purpose.
https://docs.lineapy.org/en/latest/autogen/lineapy.api.models.html#lineapy.api.models.linea_artifact.LineaArtifact.name